Add setting to allow control over telemetry event writing (#777)
microsoft/winget-cli: doc/Settings.md

- retro
- rainbow

## Telemetry

The `telemetry` settings control whether winget writes ETW events that may be sent to Microsoft on a default installation of Windows.

> Details on telemetry can be found [here](/README.md#datatelemetry), and our primary privacy statement can be found [here](/privacy.md).

Reviewer: Please don't linkify `here`. See [F84: Failure of Success Criterion 2.4.9 due to using a non-specific link such as "click here" or "more" without a mechanism to change the link text to specific text.
](https://www.w3.org/TR/WCAG20-TECHS/F84.html)
```suggestion
See [details on telemetry](../README.md#datatelemetry), and our [primary privacy statement](/privacy.md).
```

(And can you add the missing newline character to `.github/actions/spelling/patterns.txt` while you're updating this PR?)

Author: Thanks for pointing this out, and yes, I will add the newline.
